docsite-css/documentation.css

```diff
@@ -65,36 +68,48 @@
 }
 
 .component__header{
-    font-size: 4rem;
+    font-size: 3rem;
     text-transform: capitalize;
-    font-weight: 600;
```

Comment:
Yuo can remove text-transform capitalize

Reply:
I added that for consistency's sake, so that in case I end up putting in the name in any other format, this takes care of it.